tests/pytests/functional/netapi/rest_cherrypy/conftest.py

```diff
@@ -28,8 +28,9 @@ def app(client_config, load_auth):
         )
 
 
+# The order of these fixtures matter, app before io_loop
```

Comment:
A less brittle solution is to have the app fixture depend on the ioloop fixture.